docs/source/backends/openwrt.rst

```diff
@@ -1092,9 +1149,14 @@ Will result in::
             option ifname 'wlan1'
             option proto 'none'
 
-    config interface 'br_wds'
+    config device 'device_wds_bridge'
+            option name 'br-wds'
+            list ports 'wlan0'
+            list ports 'wlan1'
+            option type 'bridge'
+
+    config interface 'wds_bridge'
             option ifname 'wlan0 wlan1'
-            option network 'wds_bridge'
             option proto 'dhcp'
             option type 'bridge'
```

Comment:
`option network 'wds_bridge'` is not present even in the older version of netjsonconfig.